src/Microsoft.Identity.Client/Instance/B2COpenIdConfigurationEndpointManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.AppConfig;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.Instance
+{
+    internal class B2COpenIdConfigurationEndpointManager : IOpenIdConfigurationEndpointManager
+    {
+        private const string OpenIdConfigurationEndpoint = "v2.0/.well-known/openid-configuration";
```

Comment:
as mentioned elsewhere, should probably be in a generic helper as it's used in many places